code/game/antagonist/antagonist_factions.dm

```diff
@@ -3,6 +3,10 @@
 	set category = "Abilities"
 	if(!M.mind)
 		return
+	for (var/obj/item/weapon/implant/loyalty/I in M)
```

Comment:
Isn't there a list of implants that could be iterated through instead of all contents?